### DISCUSSION TITLE
Max Mp3 Quality

### DISCUSSION THREAD

**Discussion by mynameisdirk** [openaudible/openaudible#1042] [Q&A] [answered]
Hey! 
Why is maximaum MP3 Quality set to 160? Wasn't it 320 before?

**Comment by openaudible** [comment, marked as answer]
We changed things around to allow both VBR and fixed rate encoding. Previously we were just using VBR, which isn't 100% compatible with all  A user did a study showing there is little advantage to 320 vs. 160 for most audiobooks. You can adjust it again if needed.
